vividus-plugin-saucelabs/src/main/resources/spring.xml

```diff
@@ -25,11 +28,19 @@
         </property>
     </bean>
 
+    <bean id="sauceLabsLinkPublisher" class="org.vividus.saucelabs.SauceLabsSessionLinkPublisher">
+        <!-- Workaround for: https://github.com/saucelabs/saucerest-java/pull/127 -->
+        <constructor-arg index="0" value="#{'${saucelabs.data-center}'.toUpperCase()}" />
```

Comment:
create `RelaxedEnumPropertyEditor` instead

Reply:
but there is a change to get null if saucelabs.data-center is empty

Reply:
please elaborate

Reply:
RelaxedEnumPropertyEditor interprets empty string as null

Reply:
okay, it turned out that we convert all empty values to nulls

Reply:
with current code Spring will also convert an empty value to null